### ISSUE TITLE
AI integrity restorer

### ISSUE THREAD

**Issue by Mrcreepssss** [yogstation13/yogstation#1975]
on being carded from the integrity restorer console the camera is still locked on the console.

**Comment by ShadowDeath6** [comment]
did you try cancel camera view in the ooc tab

**Comment by Mrcreepssss** [comment]
no, the thing is, you dont have to do that with anything else as AI so i didn't even think about trying that.

**Comment by 00aquir** [comment]
cancel camera view doesn't work..
